patchwork/app.py

```diff
@@ -232,6 +242,21 @@ def cli(
             logger.debug(traceback.format_exc())
             logger.error(f"Error running patchflow {patchflow}: {e}")
             exit(1)
+    else:
+        with logger.panel(f"Patchflow {patchflow} logs") as _:
```

Comment:
This is duplicated again, best to just reassign the `panel` var and reuse it here.